README.md

```diff
@@ -1,39 +1,24 @@
-# Introduction
-
-A Simple ToDo App is built using the MVC Architecture, we have also implemented "authorization" so folx can sign up, customize & personalize the app 
+# Recipe Books
 
+A simple recipe app built using the MVC Architecture, we have also implemented "authorization" so folx can sign up, customize & personalize the app 
 ---
 
-> Be sure to add that lovely star 😀 and fork it for your own copy
-
----
-
-# Objectives
-
-- It's a beginner level app created to understand how MVC concept and logins are added
+[Link to project](#)
 
+![Anime-Gen](https://github.com/specialyas/recipebooks/blob/main/story.png?raw=true)
 ---
 
 # Who is this for? 
 
-- It's for beginners & intermediates with little more experience, to help understand the various aspects of building a node app with some complex features
+- It's for people who are looking for a place to store their recipes or just view recipes stored by others.
 
 ---
+# How it's Made
 
-# Packages/Dependencies used 
+**Tech used:** HTML, CSS, JavaScript, Express, Node and MongoDB
 
-bcrypt, connect-mongo, dotenv, ejs, express, express-flash, express-session, mongodb, mongoose, morgan, nodemon, passport, passport-local, validator
 
 ---
 
-# Development Setup

```

Comment:
I would leave lines 29 thru 39 in the README, this way people can see what to do if they want to work on the project.